src/Symfony/Bundle/FrameworkBundle/Controller/RedirectController.php

```diff
@@ -46,26 +46,23 @@ public function __construct(UrlGeneratorInterface $router = null, int $httpPort
      * In case the route name is empty, the status code will be 404 when permanent is false
      * and 410 otherwise.
      *
-     * @param Request    $request           The request instance
-     * @param string     $route             The route name to redirect to
-     * @param bool       $permanent         Whether the redirection is permanent
-     * @param bool|array $ignoreAttributes  Whether to ignore attributes or an array of attributes to ignore
-     * @param bool       $keepRequestMethod Wheter redirect action should keep HTTP request method
```

Comment:
Should probably be fixed on 2.7 instead, see https://github.com/symfony/symfony/pull/25859 in progress